crates/bindings-csharp/Codegen.Tests/fixtures/client/client.csproj

```diff
@@ -3,14 +3,8 @@
   <PropertyGroup>
     <!-- We want to ensure that BSATN.Codegen on its own produces code compatible with Unity. -->
     <!-- This means limiting project to .NET Standard 2.1 and C# 9. -->
-    <TargetFrameworks>netstandard2.1</TargetFrameworks>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
How are we testing against multiple dotnet language and runtime versions, out of curiosity?

Reply:
We don't really, we have only one specific target we support on module SDK side (only .NET 8) and on client SDK side (only .NET Standard 2.1 for Unity compat).